app.py

```diff
@@ -28,7 +30,17 @@ def create_app():
 
     handler = logging.StreamHandler(sys.stdout)
     app.logger.addHandler(handler)
```

Comment:
Is there any reason to leave this existing logging handler in addition to the new one? This causes each message to be printed twice to stdout, which creates a reasonable amount of noise